helpers.py

```diff
@@ -3,12 +3,12 @@
 
 
 class Key(str, Enum):
-    ATTENDEES = "dnd-bot:attendees"
-    DECLINERS = "dnd-bot:decliners"
-    DREAMERS = "dnd-bot:dreamers"
-    CANCELLERS = "dnd-bot:cancellers"
-    SKIP = "dnd-bot:skip"
-    INV = "dnd-bot:inv"
+    ATTENDEES = "dnd-bot:unlocked:attendees"
+    DECLINERS = "dnd-bot:unlocked:decliners"
+    DREAMERS = "dnd-bot:unlocked:dreamers"
+    CANCELLERS = "dnd-bot:unlocked:cancellers"
+    SKIP = "dnd-bot:unlocked:skip"
+    INV = "dnd-bot:locked:inv"
```

Comment:
I think also in the future we should have a wrapper to append `Guild.id` to each key so that a single bot and database could manage multiple servers.